config/config.exs

```diff
@@ -4,6 +4,18 @@ import Config
 # Configure logging
 config :logger, level: :info, truncate: :infinity
 
+## TODO: we might want to enable this with a CLI flag
```

Comment:
if we might enable this with a cli flag then it should probably be in runtime.ex

Reply:
Yes. I'm leaving that for another PR since it's not necessary unless deploying with Grafana.